Sources/SwifQL/Functions.swift

```diff
@@ -168,6 +168,7 @@ extension Fn {
         // MARK: MySQL
         
         public static var from_unixtime: Name = .init("FROM_UNIXTIME")
+        static var date_format: Fn.Name = .init("DATE_FORMAT")
```

Comment:
Could you please mark it `public` and change `Fn.Name` to just `Name`? 🙂

Reply:
yes, of course :)